.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: ${{ github.event.inputs.project_name }}
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+# Controls when the action will run.
+on:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      project_name:
+        description: 'The github repository name'
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      project_version:
+        description: 'Version number to be released e.g 10.0.3'
+      fancy:
+        description: 'If fancy versioning is required set it "True"'
+        default: 'False'
+      fancy_version_no:
+        description: 'If fancy versioning is True, then it should be set e.g 1'
+        default: '1'
+      microsoft_email:
+        description: 'Email to be written on changelogs'
+        default: 'gindibay@microsoft.com'
+      name:
+        description: 'Name to be written on changelogs'
+        default: 'Gurkan Indibay'
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition https://github.com/citusdata/tools.git
+
+      - name: install requirements
+        run: python -m pip install -r tools/python/requirements.txt
+
+      - name: Execute Package Properties Update
+        run: |
+              git config --global user.email "${MICROSOFT_EMAIL}"
+              git config --global user.name "${NAME}"
+              chmod +x ./tools/bash/update_package_properties.sh && ./tools/bash/update_package_properties.sh
```

Comment:
2 questions:

1. Why do we need `chmod +x` ? Shouldn't we set this file mode on the tools repo?
2. Why is this script not in a new directory and not in `package_automation`?

Reply:
I'm not sure but when checking out manually as here, there may be some permission problems. I had the problem when I want to execute "./tools/bash/update_package_properties.sh" command so that I needed to add this line.
I tried to push the permissions but I couldn't find a way to do it

Reply:
See https://github.com/citusdata/tools/pull/102 for my fix. After that one is merged and we switch back to checking out `develop` branch of tools, we will no longer need setting execution permissions here.

Reply:
Removed permission grant command as you suggested